doxygen/qlib/Doxyfile.in

```diff
@@ -843,12 +851,6 @@ HTML_COLORSTYLE_GAMMA  = 100
 
 HTML_TIMESTAMP         = YES
 
-# If the HTML_ALIGN_MEMBERS tag is set to YES, the members of classes,
-# files or namespaces will be aligned in HTML using tables. If set to
-# NO a bullet list will be used.
-
-HTML_ALIGN_MEMBERS     = YES
-
```

Comment:
this whole section was deleted, while the rest of the options was just commented, didn't you want to just commented out as well?